Tasks/PublishSymbolsV2/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 2,
     "Minor": 242,
```

Comment:
Minor version should be 243 as the latest 242 already releases

Reply:
I can not update minor to 243 as currentsprint is 242 and while building it, getting an error for version issues.